src/app/cube/content-pages/contribute-page/copy.ts

```diff
@@ -0,0 +1,162 @@
+                        is important in cybersecurity." 
+                        </p>
+                        <p>
+                        The <em>Blooming Onion Builder</em> also allows authors to map out their outcomes to 
+                        standardized curricular guidelines. A curricular guideline is a specific instructional statement created by a 
+                        recognized academic-adjacent group that focuses on a particular academic purpose. An example is applying the 
+                        <em>CAE Cyber Defense - Cybersecurity Ethics (KU3) - 2019</em> curricular guideline to our ethics outcome. This 
+                        curricular guideline is listed as, "Describe the role of cybersecurity in supporting and encouraging ethics, 
+                        as well as where cybersecurity practices can cause ethical conflicts." 
+                        </p>
+                        <p>
+                        An author can apply as many learning outcomes and curricular guidelines mapping needed to a learning object.
+                        </p>
+                        `
+                    },
+                    {
+                        question: 'Materials',
+                        answer:
+                        `
+                        <p>
+                        The materials tab covers the main curriculum content featured in a learning object. Content can 
+                        include uploaded files and folders, URL links, and additional notes. Not all learning objects need to contain 
+                        files, as some learning objects may link to an external site for curriculum instead. For example, an author 
+                        may use a URL link for further resource citations or larger files that are not uploadable. 
+                        </p>
+                        <p>
+                        If an author wants to include additional instructions, this can be done through the notes section. Notes are 
+                        not required but can be handy if an author wants a user to have further clarification about an object.
+                        </p>
+                        `
+                    },
+                    {
+                        question: 'Formatting Object Materials',
+                        answer:
+                        `
+                        <p>
+                        Learning object materials can be formatted in different ways as long as it conveys the lesson plan and its 
+                        material in an accessible format. If you are unsure how to start formatting your learning object material, 
+                        NSA Funded Curriculum on CLARK offers a convenient template for formatting materials.
+                        </p>
+                        <p>
+                        Go to <a href="https://clark.center/c/nccp">https://clark.center/c/nccp</a> and click the downloadable link under 
```

Comment:
We should provide a direct link instead of having them navigate to the nccp page first.

https://clark-file-upload-nccp.s3.us-east-2.amazonaws.com/Cybersecurity-Curriculum-Templates.zip

Reply:
Agreed.